### DISCUSSION TITLE
Print from web server to local POS printer

### DISCUSSION THREAD

**Discussion by craftyweb** [lukevp/ESC-POS-.NET#118] [Q&A]
Hi, 

my dad has a webshop for his store. While he is working he doesn't have time to check his website if there are new orders.
I would like to create a task that will be triggered when an order is completed.

This task should print the ticket. Is it possible to connect from the webserver to the local printer? 
He has an epson tm-t88vi POS printer.

Thanks!

**Comment by igorocampos** [comment]
Hi, it is possible, but I advise you not to do so, because you would need to forward the port so your external IP can be used to reach the printer.

A better design would be for a local application (connected to the printer) reaching out to your webserver, perhaps via HTTP request and "ask" if there are any new orders to be printed out.
